src/main/java/frc/robot/subsystems/drivetrain/BaseSwerveSubsystem.java

```diff
@@ -81,13 +109,33 @@ public BaseSwerveSubsystem(
 
     @Override
     public void periodic() {
+
         // Update pose estimator from swerve module states
         Rotation2d gyroAngle = getGyroHeading();
-        poseEstimator.update(
+        
+        Pose2d estimate = poseEstimator.update(
             gyroAngle,
             getModuleStates()
         );
 
+        xEntry.setValue(Units.metersToInches(estimate.getX()));
+        yEntry.setValue(Units.metersToInches(estimate.getY()));
+        thetaEntry.setValue(estimate.getRotation().getDegrees());
```

Comment:
not necessarily unique to this PR, but we should have a way to quickly turn off shuffleboard things to make sure we don't saturate our comms